aggregate_test.go

```diff
@@ -63,3 +100,51 @@ func TestAggregateWithSeedBy(t *testing.T) {
 		t.Errorf("From(%v).AggregateWithSeed()=%v expected %v", input, r, want)
 	}
 }
+
+func TestAggregateWithSeedByT_PanicWhenFunctionIsInvalid(t *testing.T) {
+	defer func() {
```

Comment:
it looks like you can make a test utility for this defer func like:

    func mustPanicWithError(t *testing.T, expectedErr string)

it would be useful to log a helpful error message with `got=%q expected=%q` format.